code/controllers/subsystem/garbage.dm

```diff
@@ -236,6 +237,7 @@ SUBSYSTEM_DEF(garbage)
 // Default implementation of clean-up code.
 // This should be overridden to remove all references pointing to the object being destroyed.
 // Return the appropriate QDEL_HINT; in most cases this is QDEL_HINT_QUEUE.
+// TODO: Move this and all datum var definitions into code/datums/datum.dm
```

Comment:
we should make it a style violation to have multiple procs with the same name like this, it's honestly a real gotcha for newbs